src/extra/gpkg.jl

```diff
@@ -0,0 +1,215 @@
+UNKNOWN_SRID = -2
+DEFAULT_SRID = 0
+
+# Requirement 2: contains "GPKG" in ASCII in "application_id" field of SQLite db header
+# Reminder: We have to set this (on-write) after there's some content
+# so the database file is not zero length
+
+const GP10_APPLICATION_ID::UInt32 = 0x47503130
+const GP11_APPLICATION_ID::UInt32 = 0x47503131
+const GPKG_APPLICATION_ID::UInt32= 0x47504B47
+const GPKG_1_2_VERSION::UInt32= 10200
+const GPKG_1_3_VERSION::UInt32= 10300
+const GPKG_1_4_VERSION::UInt32= 10400
```

Comment:
We don't usually annotate variables in Julia. The compiler can infer the type from the RHS. If you need to convert an `Int` literal like `10200` to `UInt32`, please do in the RHS with `UInt32(10200)`.